scripts/deploy_deps.js

```diff
@@ -11,11 +11,16 @@ module.exports = async (
 ) => {
   const UniswapAdapter = artifacts.require('UniswapAdapter')
 
+  const uniswapFactories = {
```

Comment:
I'd suggest moving these to other files since they are part of config. Maybe makes sense to put them into `deploy.mainnet.json` .. or create our own `conf.mainnet.json`?